app/views/viewer/auth.html.erb

```diff
@@ -5,6 +5,8 @@
 <div id="loginContainer">
   <% if @charge_manager.eligible? %>
     <%= link_to 'Princeton Users: Log In to Digitally Check Out', main_app.new_user_session_path, class: 'btn btn-primary', id: "login" %>
+  <% elsif restricted_viewership? %>
+    <%= link_to 'Authorized Users: Log In to View Restricted Material', main_app.new_user_session_path, class: 'btn btn-primary', id: "login" %>
```

Comment:
Special Collections asked for the text to be "Access to this material is limited to specific classes. Use your Princeton credentials to login."